README.md

```diff
@@ -0,0 +1,19 @@
+# JSON:API OpenAPI Generator
+
```

Comment:
Also, since this script doesn't support node 10, I'd recommend adding a version badge to the README.